src/docker/10-debianconsole/.prebuild.sh

```diff
@@ -1,6 +1,15 @@
 #!/bin/bash
 set -e
 
+# FIXME Remove this when Docker supplies multi-arch base images
+if [ "$ARCH" == "arm" ]; then
+  BASE_IMAGE=debian:jessie
+  ARM_BASE_IMAGE=armhf/debian:jessie
```

Comment:
For RPi 1s we have to use `armel/debian` because debians armhf is ARMv7 where as RPi 1s are ARMv6.  Now I don't think an `armel/ubuntu` exists so the Ubuntu console might never work on a RPi 1.  Probably not a big loss.